docs/sql/postgresql/migration_1.5.3_1.6.0.sql

```diff
@@ -0,0 +1,25 @@
+-- Changeset powerauth-java-server/1.6.x/20231018-add-constraint-operation-template-name.xml::1::Jan Pesek
+-- Add unique constraint to pa_operation_template.template_name
+ALTER TABLE pa_operation_template ADD CONSTRAINT pa_operation_template_template_name_uk UNIQUE (template_name);
+
+-- Changeset powerauth-java-server/1.6.x/20231103-add-activation-name-history.xml::1::Lubos Racansky
+-- Add activation_name column to pa_activation_history
+ALTER TABLE pa_activation_history ADD activation_name VARCHAR(255);
```

Comment:
This change is not mentioned in migration guide 👀

Reply:
A nice catch, I will add it.